### ISSUE TITLE
[Sourcing][P2][Accessibility] Value Help Dialog: Selected token is not read by screen reader

### ISSUE THREAD

**Issue by oscargong** [SAP/fundamental-ngx#11408]
#### Is this a bug, enhancement, or feature request?
bug / enhancement

#### Describe your proposal.

Selected token is not read by screen reader	when accessing the multi-input field 
 
- eg. we should have one stop here, and reading out "Selected Items and Conditions 2 Name 1  Name 2 "
  - ![Oscar ScreenCapture 2024-03-12 at 02 43](https://github.com/SAP/fundamental-ngx/assets/7035787/a69d6b86-386c-40ed-bc6b-39914b97c3ac)

Only the side blank can be focus
- The token should be focusable too.

Token cannot be deselected from keyboard
- we also cannot deselected a row from table

<img width="1249" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/7035787/fad4b7d3-31cc-4532-b209-7d246392bc03">

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
- this is happening on V0.48.4 
- but we are using 0.43 on our application

**Comment by droshev** [comment]
Most of the fields are not filled.

**Comment by droshev** [comment]
@oscargong can you split into separate issues?

**Comment by mikerodonnell89** [comment]
@oscargong I don't think the selected tokens are supposed to be read out loud when the input part of the multi input is accessed. Thoughts @nikolay-kolarov? Looking at ui5, they aren't doing that either. However I can open a PR to make the table and multi input tokens keyboard friendly

**Comment by oscargong** [comment]
Hi @mikerodonnell89 

- We can continue to maintain the same behavior as UI5. 
- Add keyboard support would be great!

Thanks for help!

**Comment by droshev** [comment]
@oscargong I see that the field under the table [here](https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/value-help-dialog) is accessible via the keyboard. The tokens can be focuses, and they can be focused using the keyboard.
